Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -925,6 +924,13 @@ - (void)cancel
 }
 #endif
 
+#if ENABLE(WEB_AUTHN)
+static std::span<const uint8_t> asUInt8Span(NSData* data)
```

Comment:
Where is this used?

Reply:
It's used twice below.  It used to be in SpanCocoa.h but this was the only place it's used, so I moved I there.